treeherder/etl/common.py

```diff
@@ -1,15 +1,22 @@
 import calendar
+import os
 
 import newrelic.agent
 import requests
 from dateutil import parser
 from django.conf import settings
 
+# For local development you can use a token to avoid the rate limting
+GITHUB_TOKEN = os.environ.get("GITHUB_TOKEN")
+
 
 def make_request(url, method='GET', headers=None, timeout=30, **kwargs):
     """A wrapper around requests to set defaults & call raise_for_status()."""
     headers = headers or {}
     headers['User-Agent'] = 'treeherder/{}'.format(settings.SITE_HOSTNAME)
+    if url.find("api.github.com") > -1:
+        if GITHUB_TOKEN:
+            headers["Authorization"] = "token {}".format(GITHUB_TOKEN)
```

Comment:
So you'll add the `GITHUB_CLIENT_ID:GITHUB_CLIENT_SECRET` header in a separate pr?

Reply:
Correct. I don't want to mingled more things in this PR.

Reply:
I forgot to ask, if a user has 2FA enabled does this token in the header still work?

Reply:
It does not change anything. I have 2FA enabled.